examples/formatting/profile.json

```diff
@@ -0,0 +1,62 @@
+{
+  "schemaVersion": "0.7",
+  "fields": [
+    {
+      "name": "decimal",
+      "type": "decimal",
+      "nullable": false,
+      "formatting": "£%.2f",
```

Comment:
we can't actually format numerics, i'd say that in the docs rather than pretend we can